Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/model/builder/impl/dq/DocumentMetadataResultModelBuilderImpl.java

```diff
@@ -0,0 +1,152 @@
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        String repositoryId = helper.getSingleSlotValue(ResponseSlotName.repositoryUniqueId, extrinsicObject);
+        result.setRepositoryId(repositoryId);
+
+        // classifications
+        String documentTypeCode = helper.getClassificationValue(ClassificationScheme.typeCode, extrinsicObject);
+        result.setDocumentTypeCode(documentTypeCode);
+        String documentClassCode = helper.getClassificationValue(ClassificationScheme.classCode, extrinsicObject);
+        result.setDocumentClassCode(documentClassCode);
+        String documentFormatCode = helper.getClassificationValue(ClassificationScheme.formatCode, extrinsicObject);
+        result.setDocumentFormatCode(documentFormatCode);
+
+        // author
+        RegistryObjectType author = helper.getClassification(ClassificationScheme.Author, extrinsicObject);
+        String authorPerson = helper.getSingleSlotValue("authorPerson", author);
+        result.setAuthorPerson(authorPerson);
+        String authorInstitution = helper.getSingleSlotValue("authorInstitution", author);
+        result.setAuthorInstitution(authorInstitution);
+
+        // external ids
+        String documentId = helper.getExternalIdentifierValue(IdentificationScheme.uniqueId, extrinsicObject);
+        result.setDocumentId(documentId);
+
+        // extrinsic object
+        String availabilityStatus = helper.getStatus(extrinsicObject);
+        result.setAvailabilityStatus(availabilityStatus);
+        String documentTitle = helper.getTitle(extrinsicObject);
+        result.setDocumentTitle(documentTitle);
+    }
+
+    @Override
+    public DocumentMetadataResult getDocumentMetadataResult() {
+        return result;
+    }
+
+    @Override
+    public void setRegistryObjectType(ExtrinsicObjectType extrinsicObject) {
+        this.extrinsicObject = extrinsicObject;
+    }
+
+    /**
+     * Format date time pretty.
+     *
+     * @param date the date
+     * @return the string
+     * @throws ParseException the parse exception
+     */
+    private String formatDateTimePretty(String date) throws ParseException {
+        String prettyDateTime = null;
+        if (!StringUtils.isBlank(date)) {
+            Date dDate = new SimpleDateFormat("yyyyMMddHHmmss").parse(date);
+            prettyDateTime = new SimpleDateFormat("M dd, yyyy, HH:mm Z").format(dDate);
+        }
+        return prettyDateTime;
+    }
+
+    /**
+     * Format date pretty.
+     *
+     * @param date the date
+     * @return the string
+     * @throws ParseException the parse exception
+     */
+    private String formatDatePretty(String date) throws ParseException {
+        String prettyDate = null;
+        if (!StringUtils.isBlank(date)) {
+            Date dDate = new SimpleDateFormat("yyyyMMdd").parse(date);
+            prettyDate = new SimpleDateFormat("dd/MMMM/yyyy").format(dDate);
```

Comment:
All the formaDatePretty and FormatDateTimePretty will be removed as disussed by the team. The creationDate, serviceStartTime and ServiceStopTime will be java.util.Date type instead of String